airflow/providers/amazon/aws/operators/glue.py

```diff
@@ -144,6 +149,20 @@ def execute(self, context: Context):
             self.wait_for_completion,
         )
         glue_job_run = glue_job.initialize_job(self.script_args, self.run_job_kwargs)
+        glue_job_run_url = (
+            f"{BASE_AWS_CONSOLE_LINK}/gluestudio/home?"
+            + f"region={glue_job.conn_region_name}#/job/{urllib.parse.quote(self.job_name, safe='')}/run/"
+            + glue_job_run["JobRunId"]
+        )
```

Comment:
Thanks for the PR!

Could you not reuse the format string from the Link class so as to not duplicate it here? The two could easily get out of sync. Something like (NOTE: untested):

```suggestion
        glue_job_run_url = GlueJobRunDetailsLink.format_str.format(
            region_name=glue_job.conn_region_name,
            job_name=urllib.parse.quote(self.job_name, safe=''),
            job_run_id=glue_job_run["JobRunId"])
```

Reply:
Thanks @o-nikolas for the suggestion! We should consider having a similar signature of `.persist` to also get the formatted link. My approach seems a bit ugly and forced.

Reply:
It may be an interesting PR to work for 😄

Reply:
Why not? When I initially create BaseAwsLink I've just grab an idea from BaseGoogleLink